bundles/io/org.eclipse.smarthome.io.json/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,14 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Eclipse SmartHome JSON Data Binding Abstraction Layer
+Bundle-SymbolicName: org.eclipse.smarthome.io.json
+Bundle-Version: 0.10.0.qualifier
+Automatic-Module-Name: org.eclipse.smarthome.io.json
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Import-Package: com.google.gson,
+ org.eclipse.jdt.annotation;resolution:=optional,
+ org.slf4j
+Service-Component: OSGI-INF/*.xml
+Bundle-ActivationPolicy: lazy
```

Comment:
Remove that line

Reply:
Which line, `Bundle-ActivationPolicy: lazy`? I see on #1912 it was discussed back in 2016, now I see it's no longer generating warnings, so it's removed.